content/en/docs/faq/acme.md

```diff
@@ -148,6 +148,8 @@ cert-manager will check if a DNS record has been propagated by querying the clus
 cert-manager by default uses SOA (Start of Authority) records to determine which zone name to use at your DNS provider.
 Some DNS resolvers will filter this information, if this is the case cert-manager cannot determine the zone and it is advised to [change the DNS server for DNS01 self-checks](../../configuration/acme/dns01/#setting-nameservers-for-dns01-self-check).
 
+If using dnsmasq as your local server this may occur if you use the flag `--filterwin2k`, the OpenWRT/LuCi enables this option if you check "filter useless queries". By enabling this flag, dnsmasq drops all SOA records.
```

Comment:
Maybe link to documentation for `--filterwin2k` and link to documentation for "filter useless queries".